src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -522,7 +519,7 @@ public void destroy() {
     public Set<String> getExternalResourceLocations() {
         final Set<String> ordinaryChecksResources = getExternalResourceLocations(ordinaryChecks);
         final Set<String> commentChecksResources = getExternalResourceLocations(commentChecks);
-        final int resultListSize = ordinaryChecksResources.size() + commentChecksResources.size();
+        final int resultListSize = commentChecksResources.size() + ordinaryChecksResources.size();
```

Comment:
yes, it kills [this](https://nimfadora.github.io/pr/com.puppycrawl.tools.checkstyle/TreeWalker.java.html#org.pitest.mutationtest.report.html.SourceFile@713c9199_525) mutation